webplugin/js/app/km-utils.js

```diff
@@ -257,10 +257,7 @@ KommunicateUtils = {
                 : cookiePrefix + cookie.name;
         var value = cookie.value;
         var path = '/';
-        var secure =
-            typeof cookie.secure == 'undefined'
-                ? this.isHttpsEnabledConnection()
-                : cookie.secure;
+        var secure =true;
```

Comment:
is there any issue with the customer's site itself is in HTTP, will everything work fine for them?

Reply:
i have changed code to their previous state

Reply:
this will add attribute secure over HTTPS and this samsite attribute of cookies will be strict